substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/c/codegen/CCompilerInvoker.java

```diff
@@ -124,15 +124,33 @@ protected InputStream getCompilerErrorStream(Process compilingProcess) {
         @Override
         protected CompilerInfo createCompilerInfo(Path compilerPath, Scanner scanner) {
             try {
+                String targetArch = null;
                 /* For cl.exe the first line holds all necessary information */
-                scanner.findInLine("Microsoft.?\\(R\\) C/C\\+\\+ Optimizing Compiler Version ");
+                // Checkstyle: stop
+                String forToken = "用于";
```

Comment:
Should we encode these with `\u` to avoid encoding issues somewhere down the line?